exonum-java-binding-core/rust/integration_tests/tests/service_proxy.rs

```diff
@@ -1,32 +1,47 @@
+extern crate exonum_testkit;
 extern crate integration_tests;
 extern crate java_bindings;
 #[macro_use]
 extern crate lazy_static;
+#[macro_use]
+extern crate serde_derive;
+
+use std::{
+    panic::{catch_unwind, AssertUnwindSafe},
+    sync::Arc,
+};
 
-use integration_tests::mock::service::ServiceMockBuilder;
-use integration_tests::mock::transaction::{create_mock_transaction, INFO_VALUE};
-use integration_tests::test_service::{
-    create_test_map, create_test_service, INITIAL_ENTRY_KEY, INITIAL_ENTRY_VALUE,
+use exonum_testkit::TestKitBuilder;
+
+use integration_tests::{
+    mock::{
+        service::ServiceMockBuilder,
+        transaction::{create_mock_transaction, INFO_VALUE},
+    },
+    test_service::{create_test_map, create_test_service, INITIAL_ENTRY_KEY, INITIAL_ENTRY_VALUE},
+    vm::create_vm_for_tests_with_fake_classes,
+};
+use java_bindings::{
+    exonum::{
+        blockchain::Service,
+        crypto::hash,
+        encoding::Error as MessageError,
+        messages::RawTransaction,
+        storage::{Database, MemoryDB},
+    },
+    jni::{objects::JObject, JavaVM},
+    serde_json::{from_str, Value},
+    utils::{any_to_string, convert_to_string, unwrap_jni},
+    JniExecutor, MainExecutor,
 };
-use integration_tests::vm::create_vm_for_tests_with_fake_classes;
-use java_bindings::exonum::blockchain::Service;
-use java_bindings::exonum::crypto::hash;
-use java_bindings::exonum::encoding::Error as MessageError;
-use java_bindings::exonum::messages::RawTransaction;
-use java_bindings::exonum::storage::{Database, MemoryDB};
-use java_bindings::jni::JavaVM;
-use java_bindings::serde_json::Value;
-use java_bindings::utils::any_to_string;
-use java_bindings::MainExecutor;
-use std::panic::{catch_unwind, AssertUnwindSafe};
-use std::sync::Arc;
 
 lazy_static! {
     static ref VM: Arc<JavaVM> = create_vm_for_tests_with_fake_classes();
     pub static ref EXECUTOR: MainExecutor = MainExecutor::new(VM.clone());
 }
 
 const EXCEPTION_CLASS: &str = "java/lang/RuntimeException";
+const TEST_EXCEPTION_CLASS: &str = "com/exonum/binding/fakes/mocks/TestException";
```

Comment:
Unfortunately this constant is almost useless - it is used only once, and it duplicates line 214 (with `should_panic`) - but we can't reuse it there

Reply:
Yes, but anyways to me it looks more consistent than specifying it once in that place. And yes, we definitely cannot reuse it in `should_panic`, unfortunately.